core/lib/contracts/src/lib.rs

```diff
@@ -339,14 +345,27 @@ impl BaseSystemContracts {
         BaseSystemContracts {
             bootloader,
             default_aa,
+            evm_simulator: None,
         }
     }
-    // BaseSystemContracts with proved bootloader - for handling transactions.
+
+    /// BaseSystemContracts with proved bootloader - for handling transactions.
     pub fn load_from_disk() -> Self {
         let bootloader_bytecode = read_proved_batch_bootloader_bytecode();
         BaseSystemContracts::load_with_bootloader(bootloader_bytecode)
     }
 
+    /// Loads the latest EVM simulator for these base system contracts. Logically, it only makes sense to do for the latest protocol version.
```

Comment:
Will this comment be misleading if we have multiple versions with a EVM simulator in the future?

Reply:
This comment will be updated (at least that's the idea). Once we have stable EVM simulators, they would be loaded in the new "historic" `BaseSystemContracts` constructors, and in the "default" constructor.